Specs/Core/CameraLimiterSpec.js

```diff
@@ -0,0 +1,142 @@
+ defineSuite([
+        'Core/CameraLimiter',
+        'Core/AxisAlignedBoundingBox',
+        'Core/BoundingSphere',
+        'Core/Cartesian3',
+        'Core/Matrix3',
+        'Core/OrientedBoundingBox'
+    ], function(
+        CameraLimiter,
+        AxisAlignedBoundingBox,
+        BoundingSphere,
+        Cartesian3,
+        Matrix3,
+        OrientedBoundingBox) {
+    'use strict';
+
+    var limiter;
+
+    beforeEach(function() {
+        limiter = new CameraLimiter();
+    });
+
+     it('constructing with empty constructor', function() {
+        limiter = new CameraLimiter();
+        expect(limiter).toBeDefined();
+        expect(limiter.boundingObject).not.toBeDefined();
+     });
+
+     it('constructing with empty object in constructor', function() {
+         limiter = new CameraLimiter({});
+         expect(limiter).toBeDefined();
+         expect(limiter.boundingObject).not.toBeDefined();
+     });
+
+     it('constructing with AxisAligned bounding object in constructor', function() {
+         limiter = new CameraLimiter({
+             boundingObject : new AxisAlignedBoundingBox()
+         });
+         expect(limiter).toBeDefined();
+         expect(limiter.boundingObject).toBeInstanceOf(AxisAlignedBoundingBox);
+     });
+
+     it('constructing with BoundingSphere bounding object in constructor', function() {
+         limiter = new CameraLimiter({
+             boundingObject : new BoundingSphere()
+         });
+         expect(limiter).toBeDefined();
+         expect(limiter.boundingObject).toBeInstanceOf(BoundingSphere);
+     });
+
+     it('constructing with OrientedBoundingBox bounding object in constructor', function() {
+         limiter = new CameraLimiter({
+             boundingObject : new OrientedBoundingBox()
+         });
+         expect(limiter).toBeDefined();
+         expect(limiter.boundingObject).toBeInstanceOf(OrientedBoundingBox);
+     });
+
+     var scratchCartesian3 = new Cartesian3();
+     var secondScratchCartesian3 = new Cartesian3();
+     it('limited by AxisAligned bounding object with value outside of bounding object', function() {
```

Comment:
Move the clamping logic to the tests for the `project` functions themselves. Here, just make sure we're calling the right function if the boundingObject is defined.